bears/c_languages/CPPLintBear.py

```diff
@@ -3,6 +3,12 @@
 from coalib.settings.Setting import typed_list
 
 
+def check_invalid_config(use_spaces, indent_size):
+    if not use_spaces or indent_size != 2:
+        return False
+    return True
```

Comment:
why not raise the exception right in here? Keep cause and effect in the same place, easier to understand